woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1951,6 +1971,57 @@ def test_accessor_rename(sample_df):
     assert original_df.columns.get_loc('full_name') == new_df.columns.get_loc('age')
 
 
+def test_accessor_rename_inplace(sample_df):
+    table_metadata = {'table_info': 'this is text'}
+    id_description = 'the id of the row'
+    sample_df.ww.init(index='id',
+                      time_index='signup_date',
+                      table_metadata=table_metadata,
+                      column_descriptions={'id': id_description},
+                      semantic_tags={'age': 'test_tag'},
+                      logical_types={'age': Double})
+    original_df = sample_df.ww.copy()
+    inplace_df = sample_df.ww.copy()
+
+    if dd and isinstance(sample_df, dd.DataFrame):
+        error = 'Rename inplace not supported for Dask'
+        with pytest.raises(ValueError, match=error):
+            inplace_df.ww.rename({'age': 'birthday'}, inplace=True)
+    elif ks and isinstance(sample_df, ks.DataFrame):
+        error = 'Rename inplace not supported for Koalas'
+        with pytest.raises(ValueError, match=error):
+            inplace_df.ww.rename({'age': 'birthday'}, inplace=True)
+
+    elif isinstance(sample_df, pd.DataFrame):
```

Comment:
Small detail, but this can just be an `else` block.

Reply:
done!